server/utils/utils.ts

```diff
@@ -18,4 +22,44 @@ const convertToTitleCase = (sentence: string): string =>
 
 const hasRole = (user: Express.User, role: AuthRole): boolean => user?.userRoles.includes(role) || false
 
-export { convertToTitleCase, hasRole }
+/**
+ * Converts a SimpleDateTime display value to a JSON string format dd/mm/yyyy hh:mm
+ * @param dt: SimpleDateTime
+ */
+const simpleDateTimeToJson = (dt: SimpleDateTime): string | undefined => {
+  const { inductionDate, inductionTime } = dt
+  const momentDt = moment(
+    [inductionDate.year, inductionDate.month, inductionDate.day].join('-'),
+    ['YYYY-M-D', 'YY-M-D'],
+    true
+  )
+  if (!momentDt.isValid()) {
+    return undefined
+  }
+  let hour
+  if (inductionTime.ampm === 'pm') {
```

Comment:
Moment can probably do this for us? i.e. import into moment in the format `YYYY MM DD hh mm a`, and reformat to `YYYY-MM-DD HH:mm`? Something like this?:

```
const dateString = [inductionDate.year, inductionDate.month, inductionDate.day, inductionTime.hour, inductionTime.minute, inductionTime.ampm].join(' ')
return moment(dateString, 'YYYY MM DD hh mm a').format('YYYY-MM-DD HH:mm')
```

Reply:
I was thinking this is harder than it should be. Let me give this method a go on my branch to see if I can get it simpler.

Reply:
Yep. That is better, and more simple. Thanks